link_shortener/commands/create.py

```diff
@@ -4,6 +4,7 @@
 '''
 import os
 import hashlib
+from sqlalchemy import and_
```

Comment:
Koukám že tam sám mam chyby v řazení importů, ale můžem se jim vyhnout a pak někdy opravit ty moje co sem napáchal. 

Většinou bývají tři typy importů:

- Core knihovny pythonu (os, hashlib, math)

- Importovaný knihovny (sqlalchemy, sanic, asyncio)

- Moduly projektu (link_shorener.models.blemblem)

Jsou i takhle seřazený a oddělený mezerou, takže máš jakoby tři bloky. Taky jedna z věcí kterou je dobrý dodržovat. Není to úplná buzerace, na tohle si lidí celkem potrpí, už sem to u klienta taky musel začít psát pořádně 😄

Reply:
Ok, budem na to davat pozor. :)
bc71ae1b246429de8925eb32855aebeb1324d00e